src/profile-logic/tracks.js

```diff
@@ -476,13 +477,14 @@ export function computeGlobalTracks(
   let globalTracks: GlobalTrack[] = [];
 
   // Create the global tracks.
+  const { stringArray } = profile.shared;
```

Comment:
There's another
```
const stringTable = StringTable.withBackingArray(stringArray);
if (stringTable.hasString('CompositorScreenshot')) {
```
below in this function.

Reply:
fixed in ddb4eebb85c497c161aec7e8c6d3da21145579bf